src/test/java/org/cactoos/io/LoggingInputStreamTest.java

```diff
@@ -67,13 +68,37 @@ public void readEmptyStream() {
             this.getClass().getSimpleName()
         );
         new Assertion<>(
-            "Can't read correctly an logged empty stream",
+            "Empty stream did not return -1",
             stream::read,
-            new IsEqual<>(
-                new ByteArrayInputStream(
-                    "".getBytes()
-                ).read()
-            )
+            new IsEqual<>(-1)
+        ).affirm();
+    }
+
+    @Test
+    public void readByteByByte() {
+        final LoggingInputStream stream = new LoggingInputStream(
+            new ByteArrayInputStream(
+                new byte[] {
+                    (byte) 20,
+                    (byte) 10,
+                }
+            ),
+            this.getClass().getSimpleName()
+        );
+        new Assertion<>(
```

Comment:
@ilyakharlamov you should use `Matchers.allOf` to compose these matchers

Reply:
@llorllale I wanted to check that the sequence of bytes that is read() 20, 10, -1. Tried `Matchers.allOf(new IsEqual<>(20), new IsEqual<>(10), new IsEqual<>(-1))` but no, the behavior is different than calling assertion three times.